engine/src/main/java/org/camunda/bpm/engine/externaltask/ExternalTaskQuery.java

```diff
@@ -31,6 +32,11 @@
    */
   ExternalTaskQuery externalTaskId(String externalTaskId);
 
+  /**
+   * Only select external tasks with any of the given ids
+   */
+  ExternalTaskQuery externalTaskIdIn(Set<String> externalTaskIds);
```

Comment:
This newly introduced API method is not covered by dedicated test cases.

Reply:
Tests added 👍